.github/workflows/AutomateRelease.yml

```
@@ -294,6 +304,6 @@ jobs:
           echo "- **Release Ticket URL:** ${{ needs.create_release_ticket.outputs.release_ticket_url }}" >> $GITHUB_STEP_SUMMARY
           echo "- **GitHub Release URL:** ${{ needs.publish_release.outputs.release_url }}" >> $GITHUB_STEP_SUMMARY
           echo "- **SQS Integration Ticket URL:** ${{ needs.update-integration-tickets.outputs.sqs_ticket_url }}" >> $GITHUB_STEP_SUMMARY
-          echo "- **SQC Integration Ticket URL:** ${{ needs.update-integration-tickets.outputs.sc_ticket_url }}" >> $GITHUB_STEP_SUMMARY
+          echo "- **SonarCloud Integration Ticket URL:** ${{ needs.update-integration-tickets.outputs.sqc_ticket_url }}" >> $GITHUB_STEP_SUMMARY
```

```suggestion
          echo "- **SQC Integration Ticket URL:** ${{ needs.update-integration-tickets.outputs.sqc_ticket_url }}" >> $GITHUB_STEP_SUMMARY
```
This is reverting something you already changed no? Same below

---

Ah great, thanks for catching this! I reverted your other suggestions by mistake and fixed them in this commit: https://github.com/SonarSource/sonar-php/pull/1445/commits/4a4285616ea942c1024f54d2167e9714167c1622

but I missed this one.